New HomePage

sumateja opened the pull request:
#71

Villain45 commented:
@sumateja checkout the spacing in the tagline... add that beta tag
and also what happened to the dark mode... it is not working anymore

sumateja commented:
> @sumateja checkout the spacing in the tagline... add that beta tag and also what happened to the dark mode... it is not working anymore

The spacing issue is working for me  , let me recheck Dark mode issue this any help is appreciated.

sumateja commented:
> @sumateja checkout the spacing in the tagline... add that beta tag and also what happened to the dark mode... it is not working anymore

Added Beta, fixed Dark mode issue & added SVG icons

Suraj-Bhandarkar-S commented:
![image](https://user-images.githubusercontent.com/42907076/197987784-56ce876a-6e62-4a3b-a4ca-ccf9d68f67d2.png)
![image](https://user-images.githubusercontent.com/42907076/197987830-342e4e8b-6495-4e52-8a36-39a03e536c78.png)

Color Code needs work in both the modes and Text needs Work @heysagnik @Villain45. 

@sumateja can you share your local SS. Thanks

Suraj-Bhandarkar-S reviewed (changes requested):
Need Spacing Fix for the Quick and Easy way to record Professional videos and share them instantly.

Free Online Recorded text is close needs space

sumateja commented:
> Need Spacing Fix for the Quick and Easy way to record Professional videos and share them instantly.
> 
> Free Online Recorded text is close needs space

Lemme check the text color issue , but text spacing is fine from gitpod .
<img width="1785" alt="image" src="https://user-images.githubusercontent.com/10473928/197993654-b3782455-f581-431b-b04f-1bd3778a39e5.png">

sumateja commented:
> >

Yes , i still need 2 more PR's other than this for the fest though. All are spamming with DSA codes. But was looking for meaningful PR's. So any help will be helpfull.

sumateja commented:
> > > >
> > 
> > 
> > Yes , i still need 2 more PR's other than this for the fest though. All are spamming with DSA codes. But was looking for meaningful PR's. So any help will be helpfull.
> 
> Yeah one thing you may help is by creating a Privacy Policy.md file for this repo
Opened a new PR for privacy policy, can you guys merge this PR.

heysagnik commented:
Thank you for the PR
Other than adding Hacktoberfest accepted label we can't do more . We have to fix the bugs and improve the code quality before merging to main branch. 

2 weeks back I merged the PRs with out fixing the bugs(UI) etc. and soon I lost around 200 active users of our site .
So I don't want to do the mistake again.

sumateja commented:
> Other than adding Hacktoberfest accepted label we can't do more . We have to fix the bugs and improve the code quality before merging to main branch.

Sure , but this page still needs work on selecting Camera or Screen with Camera options which needs to be addressed in another PR or issue.
So my suggestion is to rename the recorder css & recorder.pug to new files recorder_new.css & recorder_new.pug Close this PR and add this feature and fix any issues with new PR's or issues . In this way nothing will be broken.
Also please list down what issues you observe with this PR so that i will plan to fix it. (Apart from text spacing there is none i think
)

LoganTann commented:
The new homepage does not work when starting a new recording. For my feature to port the codebase into vuejs, I will reproduce first the orignal homepage, and someone will have to do the new homepage.

Villain45 commented:
@LoganTann Sure not an issue
